ml_peg/analysis/utils/decorators.py

```diff
@@ -622,12 +627,22 @@ def build_table_wrapper(*args, **kwargs) -> dict[str, Any]:
             # Use MLIP keys from first (any) metric keys
             mlips = tuple(next(iter(results.values())).keys())
 
+            name_map = dict(mlip_name_map or {})
```

Comment:
```suggestion
            name_map = mlip_name_map if mlip_name_map else {} 
```
Probably don't need to make a copy of the dict?